examples/lock-app/efr32/src/Server.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "FreeRTOS.h"
+#include "appConfig.h"
+#include "task.h"
+#include <string.h>
+#include <sys/param.h>
+
+#include "lwip/err.h"
+#include "lwip/sockets.h"
+#include "lwip/sys.h"
+#include <lwip/netdb.h>
+
+#include <inet/IPAddress.h>
+#include <inet/InetError.h>
+#include <inet/InetLayer.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <support/CodeUtils.h>
+#include <support/ErrorStr.h>
+#include <system/SystemPacketBuffer.h>
+#include <transport/SecureSessionMgr.h>
+#include <transport/UDP.h>
+
+#include "Server.h"
+
+#include "attribute-storage.h"
+#include "chip-zcl/chip-zcl-zpro-codec.h"
+#include "gen/znet-bookkeeping.h"
+#include "util.h"
+
+using namespace ::chip;
+using namespace ::chip::Inet;
+using namespace ::chip::Transport;
+
+// Transport Callbacks
+namespace {
+
+#ifndef EXAMPLE_SERVER_NODEID
+// "nRF5"
```

Comment:
Is this comment supposed to be here? It does not help me understand the code better. Maybe needs removal or expansion.

Reply:
Added a comment as we use this hardcoded nodeID for the example right now